### DISCUSSION TITLE
Change the size of a room card

### DISCUSSION THREAD

**Discussion by carlosminordomo** [UI-Lovelace-Minimalist/UI#1164] [Q&A]
Hi, first of all thank you all for the great "theme".
I was wondering how to change the size of the whole room card: (https://ui-lovelace-minimalist.github.io/UI/usage/cards/card_room/#variables)
I am trying a lot of things but without success.

Thank you so much.

**Comment by raider206** [comment]
Hello,

you have to use vertical or horizontal stacks and put the cards inside. It depends on how many cards you put in, the card will change his size in order to adapt to the stack size.
